config/config.exs

```diff
@@ -12,7 +12,9 @@ config :elixir_boilerplate, ElixirBoilerplateWeb.Endpoint,
   pubsub_server: ElixirBoilerplate.PubSub,
   render_errors: [view: ElixirBoilerplateWeb.Errors.View, accepts: ~w(html json)]
 
-config :elixir_boilerplate, ElixirBoilerplate.Repo, start_apps_before_migration: [:ssl]
+config :elixir_boilerplate, ElixirBoilerplate.Repo,
+  migration_primary_key: [name: :id, type: :binary_id],
```

Comment:
I see that in other (private) repo of our organization we have:

```elixir
migration_primary_key: [type: :binary_id],
migration_primary_key: [name: :id, type: :uuid, default: {:fragment, "gen_random_uuid()"}]
migration_primary_key: [name: :id, type: :binary_id, default: {:fragment, "gen_random_uuid()"}]
```

and you are proposing `migration_primary_key: [name: :id, type: :binary_id],`

I think that `binary_id` is the documented one but I think we should add the default. It makes it easier to create row in a GUI like TablePlus 😉